drivers/counter/counter_dtmr_cmsdk_apb.c

```diff
@@ -63,9 +63,28 @@ static u32_t counter_dtmr_cmsdk_apb_read(struct device *dev)
 	return value;
 }
 
+static int counter_dtmr_cmsdk_apb_set_wrap_alarm(struct device *dev,
+						 u32_t ticks,
+						 counter_wrap_callback_t callback,
+						 void *user_data)
+{
+	return -ENODEV;
+}
+
+static u32_t counter_dtmr_cmsdk_apb_get_wrap(struct device *dev)
+{
+	return 0;
```

Comment:
Shouldn't this be the last value before the wrap? e.g., 0xFFFF for a 16-bit counter?

Reply:
Good catch! Thanks @MaureenHelm :+1:  Solved in the new version.